ironfish-cli/src/commands/wallet/mint.ts

```diff
@@ -96,44 +104,41 @@ export class Mint extends IronfishCommand {
     }
 
     const status = await client.getNodeStatus()
-    if (!status.content.blockchain.synced) {
-      this.log(
-        `Your node must be synced with the Iron Fish network to send a transaction. Please try again later`,
-      )
-      this.exit(1)
+    if (!flags.offline) {
+      if (!status.content.blockchain.synced) {
+        this.log(
+          `Your node must be synced with the Iron Fish network to send a transaction. Please try again later`,
+        )
+        this.exit(1)
+      }
     }
 
     let account = flags.account?.trim()
```

Comment:
After some testing, we don't need to trim flags. It looks like heroku does that automatically